exercises/forth/forth_test.go

```diff
@@ -0,0 +1,53 @@
+package forth
+
+// API:
+// func Forth([]string) ([]int, error)
+//
+
+import "testing"
+
+const targetTestVersion = 1
+
+type testCase struct {
+	description string
+	input       []string
+	expected    []int // nil slice indicates error expected.
+}
+
+func sliceEqual(a, b []int) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestForth(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+	for _, tc := range testCases {
+		v, err := Forth(tc.input)
```

Comment:
After reading #275 I am wondering which of these is preferred:
```
var err error
v, err := Forth(tc.input)
```
or instead like this:
```
v, err := Forth(tc.input)
var _ error = err
``` 
?
Seems that both use the type system to insure 2nd return value is of type error. Any preference ? Looks like the trend in most exercises is the latter form.